assets/javascripts/main.js

```diff
@@ -2,7 +2,7 @@ require([
     'utils/ajax',
     'modules/analytics/setup',
     'modules/toggle',
-    'modules/optionMirror',
+    'modules/checkout/optionMirror',
```

Comment:
since it's used only in checkout, I thought it should reside with the rest of checkout related scripts